src/plugins/chapters/index.js

```diff
@@ -161,8 +164,9 @@ const ChaptersPlugin = (function () {
     wrapper.appendChild(controlBarChapterHolder);
 
     this.chaptersTrack.addEventListener('cuechange', () => {
+      const activeCues = Array.from(this.chaptersTrack.activeCues); // Safari needs Array.from()
```

Comment:
Why? I can't say why, but in Safari:
* `track.cues_` is undefined
* `track.cues` is defined, but I then discovered that
* `track.cues.foreach` is undefined, and only
* `Array.from(track.cues).forEach` is available

Same with `activeCues/activeCues_`

<img width="1413" alt="image" src="https://github.com/cloudinary/cloudinary-video-player/assets/346835/17c1be67-e411-40f9-89d7-0771276192af">

Reply:
Super weird, i approved, it seems in Safari chaptersTrack.cues is an array-like object while in other browsers it is an array object. Anyway worth adding that as a comment, i think that explaining this makes the code much more understandable instead of just a comment saying that this is needed for Safari. 

@tsi  i would comment something like 
```
chaptersTrack.cues is an array-like object in Safari only
```